controller/api/public/stat_summary_test.go

```diff
@@ -722,11 +862,102 @@ status:
 						},
 						TimeWindow: "1m",
 					},
-					expectedResponse: GenStatSummaryResponse("emoji", "deployments", "emojivoto", 1, 2, 1),
+					expectedResponse: GenStatSummaryResponse("emoji", "deployments", "emojivoto", &PodCounts{
+						MeshedPods:  1,
+						RunningPods: 2,
+						FailedPods:  1,
+					}),
 				},
 			}
 
 			testStatSummary(t, expectations)
 		})
 	})
+
+	t.Run("Queries prometheus for authority stats", func(t *testing.T) {
+		expectations := []statSumExpected{
+			statSumExpected{
+				err: nil,
+				k8sConfigs: []string{`
+apiVersion: v1
+kind: Pod
+metadata:
+  name: emojivoto-1
+  namespace: emojivoto
+  labels:
+    app: emoji-svc
+  annotations:
+    conduit.io/proxy-version: testinjectversion
+status:
+  phase: Running
+`,
+				},
+				mockPromResponse: model.Vector{
+					genPromSample("10.1.1.239:9995", "authority", "conduit", "success"),
+				},
```

Comment:
TIOLI, since `genPromSample` doesn't generate data with `dst_*` labels, the data that it does generate isn't returned. So I think it might be clearer to not generate any prometheus data at all. The test will still pass if you change this too:

```
mockPromResponse: model.Vector{},
```

Same goes for the second test that you added too.

Reply:
ahhhh thanks!